projects/admin/src/app/circulation/patron/main/main.component.ts

```diff
@@ -146,6 +153,9 @@ export class MainComponent implements OnInit, OnDestroy {
         // null and this will cause error for navigation url construction
         this._unregisterShortcuts();
         this._registerShortcuts();
+        this._operationLogsApiService.getCheckInHistory(patron.pid, 1, 1).subscribe((result: any) => {
+          this.historyCount = this._recordService.totalHits(result.hits.total);
+        });
         this._patronService.getCirculationInformations(patron.pid).subscribe((data) => {
           this.feesTotalAmount = data.fees.engaged + data.fees.preview;
           this._parseStatistics(data.statistics || {});
```

Comment:
Is that the history statistics could come with `getCirculationInformations` api call ? With this code we send a second API call to get specific stat. Maybe the best way will be to change the way of compute the 'history' value of the `getCirculationInformations` backend API ?